osrparse/replay.py

```diff
@@ -123,7 +123,8 @@ def parse_life_bar_graph(self, replay_data):
 
     def parse_timestamp_and_replay_length(self, replay_data):
         format_specifier = "<qi"
-        (self.timestamp, self.__replay_length) = struct.unpack_from(format_specifier, replay_data, self.offset)
+        (t, self.__replay_length) = struct.unpack_from(format_specifier, replay_data, self.offset)
+        self.timestamp = datetime.datetime(1, 1, 1) + datetime.timedelta(microseconds=t/10)
```

Comment:
I'd argue that (1, 1, 1) is better since that's what the windows ticks actually start from (https://msdn.microsoft.com/en-us/library/system.datetime.ticks(v=vs.110).aspx).  
If python was to change datetime's implementation, min might change since its year is MIN_YEAR not 1.  
Then again that's not likely so I guess it doesn't really matter